carma_wm_ctrl/launch/carma_wm_broadcaster.launch

```diff
@@ -12,5 +12,8 @@
 -->
 
 <launch>
-  <node name="carma_wm_broadcaster" pkg="carma_wm_ctrl" type="carma_wm_ctrl_node"/>
+  <arg name="georeference_topic" default="georeference" doc="The topic name of the georeference that is published by map_param_loader form the vector_map.osm"/>
```

Comment:
Don't pass topic names as arguments. This is a ros anti-pattern and makes remapping the topics later very difficult.

Reply:
Instead you want to give the topic a clear name and remap it as needed.